README.md

```diff
@@ -19,10 +19,26 @@ $ conda env create -f iceroad_env.yaml
 $ conda activate iceroad
 ```
 
+
+#### Install R and the following packages 
+
+```
```

Comment:
Could also add this to the yml file?

Reply:
See other comment. But I think it would be best to keep the yml as lightweight and flexible as possible with just the python dependencies for now.